project.clj

```diff
@@ -1,58 +1,56 @@
-(def ks-version "1.3.0")
-(def tk-version "1.4.0")
-
 (defproject puppetlabs/trapperkeeper-metrics "0.4.3-SNAPSHOT"
   :description "Trapperkeeper Metrics Service"
   :url "http://github.com/puppetlabs/trapperkeeper-metrics"
 
+  :min-lein-version "2.7.1"
+
   :pedantic? :abort
 
-  :dependencies [[org.clojure/clojure "1.7.0"]
+  :parent-project {:coords [puppetlabs/clj-parent "0.2.4"]
+                   :inherit [:managed-dependencies]}
+
+  :dependencies [[org.clojure/clojure]
 
                  ;; begin version conflict resolution dependencies
-                 [clj-time "0.11.0"]
-                 [commons-codec "1.9"]
-                 [org.clojure/tools.macro "0.1.5"]
-                 [org.clojure/tools.reader "1.0.0-alpha1"]
-                 [prismatic/schema "1.1.0"]
-                 [slingshot "0.12.2"]
-                 [commons-io "2.4"]
-                 [ring/ring-servlet "1.4.0"]
+                 [clj-time]
+                 [commons-codec]
+                 [org.clojure/tools.macro]
+                 [org.clojure/tools.reader]
+                 [prismatic/schema]
+                 [slingshot]
+                 [commons-io]
+                 [ring/ring-servlet]
                  ;; end version conflict resolution dependencies
 
-                 [puppetlabs/kitchensink ~ks-version]
-                 [puppetlabs/trapperkeeper ~tk-version]
-                 [puppetlabs/ring-middleware "1.0.0"]
-
-                 [ring/ring-core "1.4.0"]
-
-                 [cheshire "5.6.1"]
-                 [org.clojure/java.jmx "0.3.1"]
-                 ;; ring-defaults brings in a bad, old version of the servlet-api, which
-                 ;; now has a new artifact name (javax.servlet/javax.servlet-api).  If we
-                 ;; don't exclude the old one here, they'll both be brought in, and consumers
-                 ;; will be subject to the whims of which one shows up on the classpath first.
-                 ;; thus, we need to use exclusions here, even though we'd normally resolve
-                 ;; this type of thing by just specifying a fixed dependency version.
-                 [ring/ring-defaults "0.1.5" :exclusions [javax.servlet/servlet-api]]
+                 [puppetlabs/kitchensink]
+                 [puppetlabs/trapperkeeper]
+                 [puppetlabs/ring-middleware]
+
+                 [ring/ring-core]
+
+                 [cheshire]
+                 [org.clojure/java.jmx]
+                 [ring/ring-defaults]
                  ;; Explicitly reference the correct servlet-api so that downstream
                  ;; projects will always get it
                  [javax.servlet/javax.servlet-api "3.1.0"]
```

Comment:
Yeah, it'd be worth a quick test to see if we could get rid of this line or at least the version number in it, but we could always come back around and deal with that later if it turns out to not be a trivial change right now.